app/javascript/components/LoggedInUser.ts

```diff
@@ -65,6 +65,9 @@ type Policies = {
   community: {
     index: boolean;
   };
+  churn?: {
```

Comment:
`ReactOnRails` always sends the `churn` policy, can be made non-optional.


```suggestion
  churn: {
```